app/models/driver.rb

```diff
@@ -0,0 +1,36 @@
+class Driver < ApplicationRecord
+  has_many :trips, dependent: :destroy
+  validates :name, presence: true
+  validates :vin, presence: true
+
+  def total_earnings
+    sum = 0
+    trips.each do |trip|
+      sum += trip.cost
```

Comment:
Without validations on the Trip model, a trip with a nil `cost` can be associated with a driver. Then, this driver iterates through all of its trips. When it executes `trip.cost`, it returns nil, and then tries to add it to `sum`, which produces an error